### ISSUE TITLE
load order issue with houston:admin

### ISSUE THREAD

**Issue by Kestanous** [matb33/meteor-collection-hooks#85]
I have no idea what is causing this but I figured I would let you know. 

If I have my .meteor/packages file like this:
```
houston:admin
matb33:collection-hooks
```
I get:
```
TypeError: Cannot call method 'insert' of undefined
```
But it works fine like this:
```
matb33:collection-hooks
houston:admin
```

I assume its an issue on Houston's side but... FYI anyway.

**Comment by matb33** [comment]
Thanks for the report. Looks like it has to do with `houston:admin` depending on `dburles:mongo-collection-instances` (https://github.com/gterrono/houston/blob/master/package.js#L29) which in turn does some stuff to `Mongo.Collection` (https://github.com/dburles/mongo-collection-instances/blob/master/mongo-instances.js#L4)

**Comment by matb33** [comment]
I think this means @dburles and I need to collaborate on a way for our collection constructor wrappers to be idempotent.

In short, after he wraps his collection constructor, I should be able to safely wrap it again, and vice versa. That way load order wouldn't matter.

I suppose an initial test would be to see if I can wrap a collection constructor multiple times using my own method.

**Comment by dburles** [comment]
hey @matb33 there were problems with earlier versions of collection-instances dropping any attached prototypes (see https://github.com/dburles/meteor-collection-helpers/issues/18#issuecomment-69138482) which have been fixed in the latest version (0.2.6). I believe if houston:admin updates its dependency on collection-instances to 0.2.6 the problem should go away.

**Comment by Kestanous** [comment]
Just so all the prices are together: https://github.com/gterrono/houston/issues/270. Like @dburles and @AlexeyMK (in the Houston issue) were saying updating collection-instances should fix it. Thanks to all involved for the feedback and awesome packages! If no one else does I will close this when the issue is resolved on the Houston side.
